### PR TITLE
button height fixed

### PR DESCRIPTION
**Pull request by SAKSHAM-SUDO**
## Purpose / Description
_Describe the problem or feature and motivation_

## Fixes
Fixes _Link to the issues._

## Approach
_How does this change address the problem?_

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration (SDK version(s), emulator or physical, etc)

## Learning (optional, can help others)
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

## Checklist
_Please, go through these checks before submitting the PR._

- [ ] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [ ] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ ] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [ ] You have commented your code, particularly in hard-to-understand areas
- [ ] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: lib/screens/FoodDetailPage.dart [hunk 1]
```diff
@@ -185,20 +185,18 @@ class _FoodDetailPageContentState extends State<FoodDetailPageContent> {
...
+        child: TextButton(
+          style: ButtonStyle(
+            backgroundColor: MaterialStateProperty.all(UniversalVariables.orangeColor),
+            shape: MaterialStateProperty.all<RoundedRectangleBorder>(RoundedRectangleBorder(borderRadius: BorderRadius.circular(30.0),)
             ),
+          ),
+          onPressed: ()=>foodDetailPageBloc.addToCart(widget.fooddata),
+          child: Text("Add To Cart",style: TextStyle(fontSize: 24.0,fontWeight: FontWeight.w500,color: UniversalVariables.whiteColor),),
+        ),
+      ),
```
Comment (Akshay0701) [inline lib/screens/FoodDetailPage.dart:199]:
```suggestion
        SizedBox(
          width: double.infinity,
          child: TextButton(
            style: ButtonStyle(
              backgroundColor: MaterialStateProperty.all(UniversalVariables.orangeColor),
              shape: MaterialStateProperty.all<RoundedRectangleBorder>(RoundedRectangleBorder(borderRadius: BorderRadius.circular(30.0),)
              ),
            ),
            onPressed: () => foodDetailPageBloc.addToCart(widget.fooddata),
            child: Text("Add To Cart",style: TextStyle(fontSize: 24.0,fontWeight: FontWeight.w500,color: UniversalVariables.whiteColor),),
          ),
        ),
```

**Review by Akshay0701 — APPROVED** [review]
Thanks, I think this pr is good to merge, but let me first test it myself....

**Comment by SAKSHAM-SUDO** [comment]
Ok

**Comment by SAKSHAM-SUDO** [comment]
Is that okay?

**Review by Akshay0701 — APPROVED** [review]
Thanks! 

merging now!

**Pull request merged**